_tags

```diff
@@ -31,6 +31,7 @@ true: annot, bin_annot
 # OASIS_STOP
 
 true: warn(+A-4-6-7-9-40-42-44-48)
+true: warn_error(+1..49)
```

Comment:
So the semantic of warn_error is "if it warns, it errors" and not "enable those warning and turns them into errors" ? This is a bit strange but ok.